client/task-list/tasks/payments/RemotePayments/components/PaymentConnect.js

```diff
@@ -103,39 +137,45 @@ export const PaymentConnect = ( {
 		},
 	} );
 
-	return (
-		<Form
-			initialValues={ getInitialConfigValues() }
+	const DefaultSettings = ( { ...props } ) => (
+		<Settings
+			fields={ fields }
+			isBusy={ isOptionsRequesting }
 			onSubmitCallback={ updateSettings }
```

Comment:
I know this is `onSubmitCallback` in form, but I'm in favor of moving this to `onSubmit`.  Relevant - https://github.com/woocommerce/woocommerce-admin/pull/6708

Reply:
Done ✅